content/general/concepts/font-awesome/terms/fa-flip/fa-flip.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'fa-flip'
+Description: 'Used to flip an icon on the horizontal or vertical axis.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'World Wide Web'
+  - 'Fonts'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`fa-flip`** classes are used to flip an icon on the horizontal or vertical axis.
```

Comment:
```suggestion
The **`fa-flip`** classes are used to set an icon on the horizontal or vertical axis.
```

Reply:
@Dusch4593, this edit does not make sense to me. I don't think it describes what this does.

Reply:
I agree. I guess I was trying to look for another word besides "flip" and avoid a self-reference in the definition.